components/calendar/src/iso.rs

```diff
@@ -456,7 +456,8 @@ impl Iso {
     }
 
     // Lisp code reference: https://github.com/EdReingold/calendar-code2/blob/1ee51ecfaae6f856b0d7de3e36e9042100b4f424/calendar.l#L1237-L1258
-    pub(crate) fn iso_from_fixed(date: i32) -> Date<Iso> {
+    pub fn iso_from_fixed(date: i32) -> Date<Iso> {
```

Comment:
Issue: I still don't see why this is failing; try to debug.

Reply:
done